messages/en.json

```diff
@@ -2,12 +2,49 @@
   "title": "MABP Brand Website EN",
   "description": "Best Brand Website in the world!",
 
-  "Index": {
-    "title": "Hello world!"
-  },
   "navbar": {
     "home": "Home",
     "aboutus": "About us",
     "blog": "Blog"
+  },
+  "Header": {
+    "title": "From stellar websites to unique products, we deliver ",
+    "accent_title": "out-of-this-world solutions",
```

Comment:
out of this world doesn't need dashes right?

Reply:
Was honestly the suggestion from chatGPT to the text xD Do you want me to delete it?

Reply:
idk as you prefer